# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/CompositorBridgeChild.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/layers/ipc/CompositorBridgeChild.cpp
--- a/gfx/layers/ipc/CompositorBridgeChild.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/layers/ipc/CompositorBridgeChild.cpp	Fri May 24 11:32:54 2019 +0000
@@ -11,7 +11,7 @@
 #include "ClientLayerManager.h"  // for ClientLayerManager
 #include "base/message_loop.h"   // for MessageLoop
 #include "base/task.h"           // for NewRunnableMethod, etc
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/dom/TabGroup.h"
 #include "mozilla/layers/CompositorManagerChild.h"
 #include "mozilla/layers/ImageBridgeChild.h"
@@ -870,10 +870,10 @@
       aAllocator->GetCompositorBackendType(),
       aAllocator->SupportsTextureDirectMapping(),
       aAllocator->GetMaxTextureSize(), aFormat, gfx::gfxVars::TileSize(),
-      aFlags, gfxPrefs::LayersTilePoolShrinkTimeout(),
-      gfxPrefs::LayersTilePoolClearTimeout(),
-      gfxPrefs::LayersTileInitialPoolSize(),
-      gfxPrefs::LayersTilePoolUnusedSize(), this));
+      aFlags, StaticPrefs::LayersTilePoolShrinkTimeout(),
+      StaticPrefs::LayersTilePoolClearTimeout(),
+      StaticPrefs::LayersTileInitialPoolSize(),
+      StaticPrefs::LayersTilePoolUnusedSize(), this));
 
   return mTexturePools.LastElement();
 }